shaded-ning19/pom.xml

```diff
@@ -23,13 +23,14 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>org.apache.apex</groupId>
-    <artifactId>apex</artifactId>
-    <version>3.3.0-incubating</version>
+    <groupId>org.apache</groupId>
+    <artifactId>apache</artifactId>
+    <version>18</version>
+    <relativePath/>
   </parent>
   <groupId>org.apache.apex</groupId>
   <artifactId>apex-shaded-ning19</artifactId>
-  <version>1.0.0</version>
+  <version>1.1.0-SNAPSHOT</version>
```

Comment:
Existence in same repository and what gets released when are separate topics. These 2 artifacts are released independently of Apex core. For building core they are like other 3rd party dependencies.